### PR TITLE
Use new fNIRS name type

### PR DESCRIPTION
**Pull request by rob-luke**
This was changed in MNE-Python in https://github.com/mne-tools/mne-python/pull/7923

### CODE REVIEW

File: mnelab/mainwindow.py [hunk 1]
```diff
@@ -360,7 +360,9 @@ def data_changed(self):
                 enabled and self.model.current["ftype"] in ["XDF", "XDFZ", "XDF.GZ"]
             )
             self.actions["convert_od"].setEnabled(
-                len(mne.pick_types(self.model.current["data"].info, fnirs="fnirs_raw"))
+                len(mne.pick_types(
+                    self.model.current["data"].info,fnirs="fnirs_cw_amplitude")
+                )
```
Comment (rob-luke) [inline mnelab/mainwindow.py:365]: Flake didn't like my first push. I tried to copy your style here. I can modify as you want.

**Comment by cbrnr** [comment]
Style seems to be OK, but the closing parenthesis looks a bit weird. I wonder why it didn't like the previous version. I'll play around with it and see if flake needs to be adapted. BTW, the errors are all unrelated (but I really need to fix them).

**Comment by rob-luke** [comment]
Yeah I panicked when I saw the errors. I think the double indented bracket looks weird too. Change as you wish and Ill follow your style in future PRs

**Comment by cbrnr** [comment]
Thanks @rob-luke!

**Pull request merged**